packages/hooks/src/useIsP2PEnabled.ts

```diff
@@ -4,15 +4,14 @@ import { useStore } from '@deriv/stores';
 
 const useIsP2PEnabled = () => {
     const { client, traders_hub } = useStore();
-    const { is_authorize, loginid, currency, is_virtual } = client;
+    const { is_authorize, loginid, is_virtual } = client;
     const { is_low_risk_cr_eu_real } = traders_hub;
     const invalidate = useInvalidateQuery();
     const { data, ...rest } = useFetch('website_status', { options: { enabled: is_authorize } });
 
-    const is_p2p_supported_currency = Boolean(
-        data?.website_status?.p2p_config?.supported_currencies.includes(currency.toLocaleLowerCase())
-    );
-    const is_p2p_enabled = is_p2p_supported_currency && !is_virtual && !is_low_risk_cr_eu_real;
+    const is_p2p_config_not_disabled = data?.website_status?.p2p_config?.disabled === 0;
+
+    const is_p2p_enabled = is_p2p_config_not_disabled && !is_virtual && !is_low_risk_cr_eu_real;
 
     // Todo: should replace with the next line instead once BE is fixed.
```

Comment:
@farhan-nurzi-deriv So this todo seems to be invalid now, Right? 🤔

Reply:
There is a BE card to separate p2p_config from website_status. I will edit the todo comment to replace it with that instead